pkg/plugin/backup_pvc_action_plugin.go

```diff
@@ -70,8 +70,17 @@ func (p *NewPVCBackupItemAction) Execute(item runtime.Unstructured, backup *vele
 		return nil, nil, errors.WithStack(err)
 	}
 
+
+
 	p.Log.Info("Claiming backup repository for snapshot")
-	repositoryParameters := make(map[string]string) // TODO: retrieve the real object store config from velero storage location
+	bslName := backup.Spec.StorageLocation
+	repositoryParameters := make(map[string]string)
+
+	err = utils.RetrieveParamsFromBSL(repositoryParameters, bslName, restConfig, p.Log)
```

Comment:
Should we add a check for local mode here? repositoryParameters can remain empty in local mode.